pyfolio/tests/test_perf_attrib.py

```diff
@@ -155,6 +155,22 @@ def test_perf_attrib_simple(self):
         pd.util.testing.assert_frame_equal(expected_exposures_portfolio,
                                            exposures_portfolio)
 
+        perf_attrib_summary, exposures_summary = create_perf_attrib_stats(
+            perf_attrib_output, exposures_portfolio
+        )
+
+        self.assertEqual(perf_attrib_summary['Annualized Specific Return'],
+                         perf_attrib_summary['Total Annualized Return'])
+
+        self.assertEqual(perf_attrib_summary['Cumulative Specific Return'],
+                         perf_attrib_summary['Total Returns'])
+
+        pd.util.testing.assert_frame_equal(
+            exposures_summary,
+            pd.DataFrame(0.0, index=exposures_summary.index,
+                         columns=exposures_summary.columns)
```

Comment:
Nice, can we just be explicit in specifying `index` and `columns`? Otherwise this will pass for any dataframe of zeros, since we're referencing the columns and index of the df being tested to define the expected

Reply:
explicitly listed index and columns